Solutions/Windows Server DNS/Data Connectors/template_ASimDnsActivityLogs.JSON

```diff
@@ -0,0 +1,80 @@
+{
+    "id": "ASimDnsActivityLogs",
+    "title": "Windows DNS Events via AMA",
+    "publisher": "Microsoft",
+    "descriptionMarkdown": "The Windows DNS log connector allows you to easily filter and stream all analytics logs from your Windows DNS servers to your Microsoft Sentinel workspace using the Azure Monitoring agent (AMA). Having this data in Microsoft Sentinel helps you identify issues and security threats such as:\n- Trying to resolve malicious domain names.\n- Stale resource records.\n- Frequently queried domain names and talkative DNS clients.\n- Attacks performed on DNS server.\n\nYou can get the following insights into your Windows DNS servers from Microsoft Sentinel:\n- All logs centralized in a single place.\n- Request load on DNS servers.\n- Dynamic DNS registration failures.\n\nWindows DNS events are supported by Advanced SIEM Information Model (ASIM) and stream data into the ASimDnsActivityLogs table. [Learn more](https://docs.microsoft.com/azure/sentinel/normalization)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "ASimDnsActivityLogs",
+            "baseQuery": "ASimDnsActivityLogs\n| where EventProduct == \"DNS Server\""
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description": "All logs",
+            "query": "ASimDnsActivityLogs\n| where EventProduct == \"DNS Server\"\n            | sort by TimeGenerated"
+        },
+        {
+            "description": "Count the number of failed DNS queries for each source IP address and failure type query.",
+            "query": "ASimDnsActivityLogs\n| where EventProduct == \"DNS Server\"\n            | where EventType == \"Query\" and EventResult == \"Failure\"\n| summarize count() by SrcIpAddr, EventResultDetails"
+        },
+        {
+            "description": "Count the number of queries that return NXDOMAIN, indicating that the queries domain name does not exist, and compares the count to a threshold of 100.",
+            "query": "ASimDnsActivityLogs\n| where EventProduct == \"DNS Server\"\n            | where EventResultDetails == \"NXDOMAIN\"\n| summarize c=count() by SrcIpAddr\n| where c > 100"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "ASimDnsActivityLogs",
```

Comment:
I am not sure how connectivityCriterias works, but if this is used to check connectivity, we need to add the condition for EventProduct and EventVendor also here.

Reply:
Hi @oshezaf, as per the discussion the connectivityCriterias will be updated at once for all connectors.

![image](https://user-images.githubusercontent.com/94349919/218477240-b273d6a0-53bb-429d-9aca-7a9cd5511c9d.png)